src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Knowncolors.cs

```diff
@@ -675,20 +666,26 @@ internal static KnownColor ColorStringToKnownColor(string colorString)
 #if !PBTCOMPILER
         internal static KnownColor ArgbStringToKnownColor(string argbString)
         {
-            string argbUpper = argbString.Trim().ToUpper(System.Globalization.CultureInfo.InvariantCulture);
+            ReadOnlySpan<char> argbSpan = argbString.AsSpan().Trim();
 
-            KnownColor color;
-            if (s_knownArgbColors.TryGetValue(argbUpper, out color))
-                return color;
+            // Use NumberStyles.AllowHexSpecifier instead of NumberStyles.HexNumber because NumberStyles.HexNumber
+            // trims the whitespaces when we already trim it in the code above and we don't consider values
+            // with whitespaces between the "#" and the hex value to be valid values.
+            if (argbSpan.StartsWith('#') && uint.TryParse(argbSpan[1..], NumberStyles.HexNumber, CultureInfo.InvariantCulture, out uint uintValue))
```

Comment:
didn't you mean to use... `NumberStyles.AllowHexSpecifier`?

Reply:
Yes thanks for noticing, I forgot to push my latest changes.